integration-tests/tests/advanced-happy-path.spec.ts

```diff
@@ -378,18 +376,18 @@ describe('Advanced Happy path', () => {
 
     it('Verify EC Integration Test pipeline runs Logs Tab', () => {
       UIhelper.clickTab('Logs');
-      DetailsTab.verifyLogs('"result": "FAILURE"');
+      DetailsTab.verifyLogs('"result": "SUCCESS"');
```

Comment:
I guess now we are not checking a Failure anywhere, right? And I assume we are OK with that, right? :smiley:

Reply:
earlier the Enterprise contract image bundle used to report failure one rule.
now we are using the git repo and in that all the rules reports Success.